packages/react-composites/src/composites/CallWithChatComposite/BasicHeader.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { CommandBarButton, Stack } from '@fluentui/react';
+import { useTheme } from '@internal/react-components';
+import React from 'react';
+import { sidePaneHeaderStyles } from '../common/styles/ParticipantContainer.styles';
+
+/**
+ * @private
+ */
+export const BasicHeader = (props: { headingText: string; onClose: () => void }): JSX.Element => {
```

Comment:
nit: Adjectives like `Basic`, `Simple` etc are a code smell.

It doesn't tell me why this is `Basic` (for that I need to know that there exists a `TabHeader`, and this is the non-tab version).

`SidePaneHeader` would be better as this is the header used by the side pane.